docs/userguide.md

```diff
@@ -7,8 +7,18 @@ is a walkthrough tutorial for how to deploy the federation v2 control plane.
 
 ### Binaries
 
-The federation deployment depends on `apiserver-boot`, `crinit`, and `kubectl`
-being installed in the path. These binaries can be installed via the
+Federation can be deployed with a prebuilt image from federation-v2 master using `scripts/deploy-federation-latest.sh`.
+The federation deployment depends on [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) being installed in the path.
+
+Alternatively, you can build your own image with `scripts/deploy-federation.sh <image> cluster2 cluster3`
+
+Where `<image>` is in the form
+`<containerregistry>/<username>/<imagename>:<tagname>` e.g.
+`docker.io/<username>/federation-v2:test`.
+
+This requires the [`kubebuilder`](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/quick_start.md#installation) binary.
```

Comment:
As above, it is better specify the kubebuilder version as well, it works with both 0.1.11 and 0.1.12, we can specify that the minimum version should be 0.1.11

Reply:
I don't think kubebuilder 0.1.11 will work because the status subresource work was released in 0.1.12.